src/frontends/pytorch/src/op/pythonop.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "pt_framework_node.hpp"
+#include "translate_session.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_pythonop(NodeContext& context) {
+    auto decoder = context.get_decoder();
+    FRONT_END_OP_CONVERSION_CHECK(decoder->get_subgraph_size() == 1,
+                                  "PythonOp must have 1 subgraph to be able to translate it to OV.");
+    auto body = context.convert_subgraph(0);
+    auto session = context.get_session();
+
+    std::map<size_t, ParameterVector> inputs_map;
+    for (const auto& param : body->get_parameters()) {
+        auto tensor_idx = session->decode_tensor_name(param->output(0));
+        if (!inputs_map.count(tensor_idx)) {
+            inputs_map[tensor_idx] = {param};
+        } else {
+            inputs_map[tensor_idx].push_back(param);
+        }
+    }
+    for (const auto& input : inputs_map) {
+        auto external_output = context.get_tensor_from_model(input.first);
+        if (external_output.get_node()) {
+            for (auto input_node : input.second) {
```

Comment:
If you have multiple nodes with same id it will replace it in sequence and result in only the last one existing. I think we need to throw exception in such cases, because it is wrong obviously.

Reply:
Added `FRONT_END_OP_CONVERSION_CHECK` not allowing multiple parameters for same input should prevent it. Removed for loop over input.second.